example.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from itertools import pairwise
+from pathlib import Path
+from typing import Any, Iterable, Iterator, Mapping, Tuple
+
+from telcell.data.models import Track
+from telcell.data.parsers import parse_measurements_csv
+from telcell.data.utils import extract_intervals, split_track_by_interval
+from telcell.models import DummyModel
+from telcell.pipeline import run_pipeline
+
+
+def dummy_cruncher(tracks: Iterable[Track]) \
+        -> Iterator[Tuple[Track, Track, Mapping[str, Any]]]:
+    """
+    Dummy data cruncher that takes each consecutive pair of tracks `(track_a,
+    track_b)` and splits `track_a` into slices for each day. Yields a 3-tuple
+    for each such slice that contains the following:
+        - A `Track` consisting of the `track_a` measurements for that day;
+        - A reference to `track_b`;
+        - A mapping with a `Track` containing all other `track_a` measurements.
+    """
+
+    for track_a, track_b in pairwise(tracks):
```

Comment:
`itertools.combinations` :clown_face:

Reply:
Moet dit voor nu aangepast worden vind je? Het is namelijk maar een voorbeeld, is een beetje om het even welke tracks je pakt denk ik?

Reply:
Oh right, `example.py`, dan maakt het niet heel veel uit, nee. We zouden er een comment aan kunnen wijden, maar verder lijkt me prima idd.

Reply:
Comment lijkt me wel het minste, want je zou mij zo in de maling kunnen nemen door iets te doen dat lijkt op wat je verwacht, maar het net niet is.

Reply:
Er staat toch al een comment boven waarin dit wordt uitgelegd?

```python
"""
Dummy data cruncher that takes each consecutive pair of tracks [...]
"""
```